dbms/src/Flash/Planner/ExecutorIdGenerator.h

```diff
@@ -43,29 +43,29 @@ class ExecutorIdGenerator
         switch (executor.tp())
         {
         case tipb::ExecType::TypeSelection:
-            return fmt::format("{}_selection", ++current_id);
+            return fmt::format("selection_{}", current_id++);
```

Comment:
changed from ++current_id to current_id++, to allow 0 for executor_id?

Reply:
> changed from ++current_id to current_id++, to allow 0 for executor_id?

Yes